flake.nix

```
@@ -112,6 +112,23 @@
             };
           };
 
+        flake.modules = {
```

We have a standard attribute for that: https://nixos.wiki/wiki/Flakes#Output_schema

```suggestion
        flake.nixosModules = {
```